piffle/iiif.py

```diff
@@ -108,3 +108,125 @@ def format(self, image_format):
         img = self.get_copy()
         img.image_options['format'] = image_format
         return img
+
+    def parse_from_url(self, url):
+        '''Parses Image API parameters from URL provided
+        Per http://iiif.io/api/image/2.0/#image-request-uri-syntax, using slashes to navigate URL'''
+        
+        url_components = url.split('/')
+        
+        _quality, _format = url_components[-1].split('.')
```

Comment:
Probably need some error handling here (and/or maybe in the init method).  What if the url is malformed somehow?  Also, what about supporting the info.json url to initialize the api endpoint and image id?